app/services/notices/setup/create-alternate-notice.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+/**
+ * Orchestrates creating a new notice and notifications for returns invitation emails that failed
+ *
+ * @module CreateAlternateNoticeService
+ */
+
+const CreateNotificationsService = require('./create-notifications.service.js')
+const DetermineNoticeTypeService = require('./determine-notice-type.service.js')
+const EventModel = require('../../../models/event.model.js')
+const FetchAlternateRecipientsService = require('./fetch-alternate-recipients.service.js')
+const FetchFailedReturnsInvitationsService = require('./fetch-failed-returns-invitations.service.js')
+
+const { timestampForPostgres } = require('../../../lib/general.lib.js')
+const { NoticeType, NoticeJourney } = require('../../../lib/static-lookups.lib.js')
+
+/**
+ * Orchestrates creating a new notice and notifications for returns invitation emails that failed
+ *
+ * @param {module:EventModel} notice - The email notice to check for failed notifications
+ *
+ * @returns {Promise<object>} - the created notifications and reference code
+ */
+async function go(notice) {
+  const { failedLicenceRefs, failedReturnIds } = await FetchFailedReturnsInvitationsService.go(notice.id)
+
+  if (failedReturnIds.length === 0) {
+    return { notice: null, notifications: [] }
+  }
+
+  const noticeType = DetermineNoticeTypeService.go(NoticeType.INVITATIONS)
+  const recipients = await FetchAlternateRecipientsService.go(failedReturnIds)
+  const letterNotice = await _notice(notice, noticeType, recipients, failedLicenceRefs)
+  const notifications = await _notifications(letterNotice, recipients)
+
+  return { notice: letterNotice, notifications }
```

Comment:
If I were going to name this var something, IMHO, `alternateNotice` is more applicable. And on the following line, we didn't name the var `letterNotifications`. 

In this context, I think you can just not bother.  

```suggestion
  const notice = await _notice(notice, noticeType, recipients, failedLicenceRefs)
  const notifications = await _notifications(letterNotice, recipients)

  return { notice, notifications }
```